Google.Api.Generator.Tests/ProtoTests/UnitTests/Testing.UnitTests/UnitTestsClient.cs

```diff
@@ -0,0 +1,272 @@
+﻿using gax = Google.Api.Gax;
```

Comment:
Do we need this file, as opposed to skipping it? Maybe I've missed something about how all of this infrastructure works.

Reply:
This file is required to allow the generated tests to execute.
Because the "expected" generated test files are present in the test project, they execute within the "dotnet test" run; which is nice, as it means we always verify that the generated tests (= expected test code) pass.